### ISSUE TITLE
How secured is et as compared to mosh?

### ISSUE THREAD

**Issue by nitin-gyany** [MisterTea/EternalTerminal#663]
I use mosh and have the scrolling problem.
I saw the "How it works" and got that it doesn't use SSH. 
Is et safe to be used on a Production server?

**Comment by MisterTea** [comment]
Yes, et has undergone several security audits and the only issues surfaced have been potential DoS attacks that were fixed. There's been no report of someone compromising an et connection.

**Comment by MisterTea** [comment]
Closing this as answered. This is a general question, not actionable issue tracking. Security architecture/audit claims should instead live in maintained documentation.
